examples/chip-tool/templates/tests/helper.js

```diff
@@ -47,8 +47,31 @@ function asPropertyValue(options)
   return name;
 }
 
+async function asDecodableType()
+{
+  const options = { 'hash' : { ns : this.cluster } };
+  let type;
```

Comment:
We seem to write to `type`, but when do we use it for anything?

Reply:
I forgot to push the last line of the method that uses `zapTypeToDecodableClusterObjectType`...